### ISSUE TITLE
Additional locks not showing up

### ISSUE THREAD

**Issue by mcraigm** [mlamoure/Indigo-August-Home#9]
I added 2 new locks (August WiFi smart lock 4 gen) to my August app. The old gen3 locks were already installed and working. The new locks do not show up.   I thought it might be because I did not have the August connect, so I added a connect to one of the new locks. Neither one is showing up.  I upgraded the plugin to .20.

**Comment by mlamoure** [comment]
I’ve had reports of the 4th gen working.  Shouldn’t need the connect.  Did you try restarting the plugin after adding them to your August keychain?

**Comment by mcraigm** [comment]
Yes, I did the reload and disabled and enabled.

**Comment by mlamoure** [comment]
Can you turn debugging on and restart the plugin.  Send me the logs.  I’ll take a look.

**Comment by mcraigm** [comment]
Hi Micheal

Thanks for your time!

 

2020-09-07 12:39:43      Application               Stopped plugin "August Home 1.0.20"

2020-09-07 12:39:47      Application               Enabling plugin "August Home 1.0.20"

2020-09-07 12:39:47      Application               Starting plugin "August Home 1.0.20" (pid 89074)

2020-09-07 12:39:49      August Home Debug               Response HTTP Status Code: 200

2020-09-07 12:39:49      August Home Debug               Response HTTP Response Body: {"installId":"cb5fdb67-a2c8-4673-b316-ca55fa8e2235","applicationId":"","userId":"677f1a24-1d34-49ae-8f9b-d15db33c6a52","vInstallId":true,"vPassword":true,"vEmail":false,"vPhone":false,"hasInstallId":true,"hasPassword":true,"hasEmail":true,"hasPhone":true,"isLockedOut":false,"captcha":"","email":[],"phone":[],"expiresAt":"2021-01-05T16:39:48.959Z","temporaryAccountCreationPasswordLink":"","iat":1599496788,"exp":1609864788,"LastName":"Mackenzie","FirstName":"Craig"}

2020-09-07 12:39:49      Application               Started plugin "August Home 1.0.20"

2020-09-07 12:39:49      August Home Debug               startup called

2020-09-07 12:39:49      August Home Debug               Completed processing activity logs

2020-09-07 12:39:49      August Home Debug               Obtaining a list of doorbells...

2020-09-07 12:39:49      August Home Debug               Response HTTP Status Code: 200

2020-09-07 12:39:49      August Home Debug               Response HTTP Response Body: {}

2020-09-07 12:39:49      August Home Debug               Created variables for lock August Back Door

2020-09-07 12:39:49      August Home Debug               Created variables for lock August Workshop Door

2020-09-07 12:39:49      August Home Debug               started variable updates

2020-09-07 12:39:49      August Home Debug               finished variable updates

2020-09-07 12:39:49      August Home Debug               deviceStartComm: August Back Door

2020-09-07 12:39:50      August Home Debug               getLockStatus Response HTTP Status Code: 200

2020-09-07 12:39:50      August Home Debug               getLockStatus Response HTTP Response Body: {"status":"unlocked","dateTime":"2020-09-07T16:24:52.945Z","isLockStatusChanged":false,"valid":true,"doorState":"closed"}

2020-09-07 12:39:50      August Home Debug               Adding new house: 109c8131-f8e1-453a-89d1-35a59d79fb09 to house cache list

2020-09-07 12:39:50      August Home Debug               Obtaining a list of doorbells...

2020-09-07 12:39:50      August Home Debug               Response HTTP Status Code: 200

2020-09-07 12:39:50      August Home Debug               Response HTTP Response Body: {}

2020-09-07 12:39:50      August Home Debug               deviceStartComm: August Workshop Door

2020-09-07 12:39:51      August Home Debug               getLockStatus Response HTTP Status Code: 200

2020-09-07 12:39:51      August Home Debug               getLockStatus Response HTTP Response Body: {"status":"unlocked","dateTime":"2020-09-07T16:24:52.945Z","isLockStatusChanged":false,"valid":true,"doorState":"closed"}

2020-09-07 12:39:51      August Home Debug               Adding new house: 109c8131-f8e1-453a-89d1-35a59d79fb09 to house cache list

2020-09-07 12:39:51      August Home Debug               Obtaining a list of doorbells...

2020-09-07 12:39:52      August Home Debug               Response HTTP Status Code: 200

2020-09-07 12:39:52      August Home Debug               Response HTTP Response Body: {}

2020-09-07 12:39:52      August Home Debug               Starting concurrent tread

2020-09-07 12:39:52      August Home Debug               Processing activity logs for house ID: 109c8131-f8e1-453a-89d1-35a59d79fb09

2020-09-07 12:39:52      August Home Debug               processing activity item: {u'info': {u'DateLogActionID': u'7DA59B8C1692437082B59DBBDA5BBB68+Mon Sep 07 2020 15:31:26 GMT+0000 (Coordinated Universal Time)unlock', u'remote': True}, u'deviceName': u'Workshop door', u'house': {u'houseName': u'Haist House', u'houseID': u'109c8131-f8e1-453a-89d1-35a59d79fb09'}, u'otherUser': {u'UserName': u'deleteduser', u'LastName': u'User', u'UserID': u'deleted', u'PhoneNo': u'deleted', u'FirstName': u'Unknown'}, u'dateTime': 1599492686287, u'entities': {u'device': u'7DA59B8C1692437082B59DBBDA5BBB68', u'house': u'109c8131-f8e1-453a-89d1-35a59d79fb09', u'callingUser': u'677f1a24-1d34-49ae-8f9b-d15db33c6a52', u'otherUser': u'deleted', u'activity': u'5f56524ebd4e2e7a7ea467bd'}, u'callingUser': {u'LastName': u'Mackenzie', u'UserID': u'677f1a24-1d34-49ae-8f9b-d15db33c6a52', u'imageInfo': {u'original': {u'url': u'https://d33mytkkohwnk6.cloudfront.net/user/4d9992a0-3278-4179-bcd9-ba3a7c193fda.jpg', u'width': 2131, u'format': u'jpg', u'secure_url': u'https://d33mytkkohwnk6.cloudfront.net/user/4d9992a0-3278-4179-bcd9-ba3a7c193fda.jpg', u'height': 2131}, u'thumbnail': {u'url': u'https://d33mytkkohwnk6.cloudfront.net/user/aff607c9-292d-4b7b-b3e3-4589acee2447.jpg', u'width': 128, u'format': u'jpg', u'secure_url': u'https://d33mytkkohwnk6.cloudfront.net/user/aff607c9-292d-4b7b-b3e3-4589acee2447.jpg', u'height': 128}}, u'FirstName': u'Craig'}, u'deviceType': u'lock', u'deviceID': u'7DA59B8C1692437082B59DBBDA5BBB68', u'action': u'unlock'}

2020-09-07 12:39:52      August Home Debug               processing activity item: {u'info': {u'DateLogActionID': u'F37A48FE11F140A4840E9552468A4945+Mon Sep 07 2020 15:26:30 GMT+0000 (Coordinated Universal Time)lock', u'remote': True}, u'deviceName': u'Front Door', u'house': {u'houseName': u'Haist House', u'houseID': u'109c8131-f8e1-453a-89d1-35a59d79fb09'}, u'otherUser': {u'UserName': u'deleteduser', u'LastName': u'User', u'UserID': u'deleted', u'PhoneNo': u'deleted', u'FirstName': u'Unknown'}, u'dateTime': 1599492390918, u'entities': {u'device': u'F37A48FE11F140A4840E9552468A4945', u'house': u'109c8131-f8e1-453a-89d1-35a59d79fb09', u'callingUser': u'677f1a24-1d34-49ae-8f9b-d15db33c6a52', u'otherUser': u'deleted', u'activity': u'5f565126e4272f3271c2763a'}, u'callingUser': {u'LastName': u'Mackenzie', u'UserID': u'677f1a24-1d34-49ae-8f9b-d15db33c6a52', u'imageInfo': {u'original': {u'url': u'https://d33mytkkohwnk6.cloudfront.net/user/4d9992a0-3278-4179-bcd9-ba3a7c193fda.jpg', u'width': 2131, u'format': u'jpg', u'secure_url': u'https://d33mytkkohwnk6.cloudfront.net/user/4d9992a0-3278-4179-bcd9-ba3a7c193fda.jpg', u'height': 2131}, u'thumbnail': {u'url': u'https://d33mytkkohwnk6.cloudfront.net/user/aff607c9-292d-4b7b-b3e3-4589acee2447.jpg', u'width': 128, u'format': u'jpg', u'secure_url': u'https://d33mytkkohwnk6.cloudfront.net/user/aff607c9-292d-4b7b-b3e3-4589acee2447.jpg', u'height': 128}}, u'FirstName': u'Craig'}, u'deviceType': u'lock', u'deviceID': u'F37A48FE11F140A4840E9552468A4945', u'action': u'lock'}

2020-09-07 12:39:52      August Home Debug               processing activity item: {u'info': {u'DateLogActionID': u'F37A48FE11F140A4840E9552468A4945+Mon Sep 07 2020 15:26:27 GMT+0000 (Coordinated Universal Time)unlock', u'remote': True}, u'deviceName': u'Front Door', u'house': {u'houseName': u'Haist House', u'houseID': u'109c8131-f8e1-453a-89d1-35a59d79fb09'}, u'otherUser': {u'UserName': u'deleteduser', u'LastName': u'User', u'UserID': u'deleted', u'PhoneNo': u'deleted', u'FirstName': u'Unknown'}, u'dateTime': 1599492387514, u'entities': {u'device': u'F37A48FE11F140A4840E9552468A4945', u'house': u'109c8131-f8e1-453a-89d1-35a59d79fb09', u'callingUser': u'677f1a24-1d34-49ae-8f9b-d15db33c6a52', u'otherUser': u'deleted', u'activity': u'5f5651230ba7806e5bd63a9a'}, u'callingUser': {u'LastName': u'Mackenzie', u'UserID': u'677f1a24-1d34-49ae-8f9b-d15db33c6a52', u'imageInfo': {u'original': {u'url': u'https://d33mytkkohwnk6.cloudfront.net/user/4d9992a0-3278-4179-bcd9-ba3a7c193fda.jpg', u'width': 2131, u'format': u'jpg', u'secure_url': u'https://d33mytkkohwnk6.cloudfront.net/user/4d9992a0-3278-4179-bcd9-ba3a7c193fda.jpg', u'height': 2131}, u'thumbnail': {u'url': u'https://d33mytkkohwnk6.cloudfront.net/user/aff607c9-292d-4b7b-b3e3-4589acee2447.jpg', u'width': 128, u'format': u'jpg', u'secure_url': u'https://d33mytkkohwnk6.cloudfront.net/user/aff607c9-292d-4b7b-b3e3-4589acee2447.jpg', u'height': 128}}, u'FirstName': u'Craig'}, u'deviceType': u'lock', u'deviceID': u'F37A48FE11F140A4840E9552468A4945', u'action': u'unlock'}

2020-09-07 12:39:52      August Home Debug               processing activity item: {u'eventID': u'4639ce3d-c76c-4934-aaa8-b63058397cf9', u'deviceName': u'Front Door', u'house': {u'houseName': u'Haist House', u'houseID': u'109c8131-f8e1-453a-89d1-35a59d79fb09'}, u'otherUser': {u'UserName': u'deleteduser', u'LastName': u'User', u'UserID': u'deleted', u'PhoneNo': u'deleted', u'FirstName': u'Unknown'}, u'dateTime': 1599492350308, u'entities': {u'device': u'F37A48FE11F140A4840E9552468A4945', u'house': u'109c8131-f8e1-453a-89d1-35a59d79fb09', u'callingUser': u'677f1a24-1d34-49ae-8f9b-d15db33c6a52', u'otherUser': u'deleted', u'activity': u'5f5650feb361e4ebefccc9e1'}, u'callingUser': {u'LastName': u'Mackenzie', u'UserID': u'677f1a24-1d34-49ae-8f9b-d15db33c6a52', u'imageInfo': {u'original': {u'url': u'https://d33mytkkohwnk6.cloudfront.net/user/4d9992a0-3278-4179-bcd9-ba3a7c193fda.jpg', u'width': 2131, u'format': u'jpg', u'secure_url': u'https://d33mytkkohwnk6.cloudfront.net/user/4d9992a0-3278-4179-bcd9-ba3a7c193fda.jpg', u'height': 2131}, u'thumbnail': {u'url': u'https://d33mytkkohwnk6.cloudfront.net/user/aff607c9-292d-4b7b-b3e3-4589acee2447.jpg', u'width': 128, u'format': u'jpg', u'secure_url': u'https://d33mytkkohwnk6.cloudfront.net/user/aff607c9-292d-4b7b-b3e3-4589acee2447.jpg', u'height': 128}}, u'FirstName': u'Craig'}, u'info': {u'action': u'lock', u'dateTime': u'2020-09-07T15:25:50.308Z', u'DateLogActionID': u'F37A48FE11F140A4840E9552468A4945+Mon Sep 07 2020 15:25:50 GMT+0000 (Coordinated Universal Time)lock', u'crypto': u'success', u'mechanical': u'success'}, u'deviceType': u'lock', u'deviceID': u'F37A48FE11F140A4840E9552468A4945', u'action': u'lock'}

2020-09-07 12:39:52      August Home Debug               processing activity item: {u'eventID': u'180d3bdc-60fb-4b54-b3ee-689922dc77ef', u'deviceName': u'Front Door', u'house': {u'houseName': u'Haist House', u'houseID': u'109c8131-f8e1-453a-89d1-35a59d79fb09'}, u'otherUser': {u'UserName': u'deleteduser', u'LastName': u'User', u'UserID': u'deleted', u'PhoneNo': u'deleted', u'FirstName': u'Unknown'}, u'dateTime': 1599492347128, u'entities': {u'device': u'F37A48FE11F140A4840E9552468A4945', u'house': u'109c8131-f8e1-453a-89d1-35a59d79fb09', u'callingUser': u'677f1a24-1d34-49ae-8f9b-d15db33c6a52', u'otherUser': u'deleted', u'activity': u'5f5650fbe4272f0361c26aa0'}, u'callingUser': {u'LastName': u'Mackenzie', u'UserID': u'677f1a24-1d34-49ae-8f9b-d15db33c6a52', u'imageInfo': {u'original': {u'url': u'https://d33mytkkohwnk6.cloudfront.net/user/4d9992a0-3278-4179-bcd9-ba3a7c193fda.jpg', u'width': 2131, u'format': u'jpg', u'secure_url': u'https://d33mytkkohwnk6.cloudfront.net/user/4d9992a0-3278-4179-bcd9-ba3a7c193fda.jpg', u'height': 2131}, u'thumbnail': {u'url': u'https://d33mytkkohwnk6.cloudfront.net/user/aff607c9-292d-4b7b-b3e3-4589acee2447.jpg', u'width': 128, u'format': u'jpg', u'secure_url': u'https://d33mytkkohwnk6.cloudfront.net/user/aff607c9-292d-4b7b-b3e3-4589acee2447.jpg', u'height': 128}}, u'FirstName': u'Craig'}, u'info': {u'action': u'unlock', u'dateTime': u'2020-09-07T15:25:47.128Z', u'DateLogActionID': u'F37A48FE11F140A4840E9552468A4945+Mon Sep 07 2020 15:25:47 GMT+0000 (Coordinated Universal Time)unlock', u'crypto': u'success', u'mechanical': u'success'}, u'deviceType': u'lock', u'deviceID': u'F37A48FE11F140A4840E9552468A4945', u'action': u'unlock'}

2020-09-07 12:39:52      August Home Debug               processing activity item: {u'eventID': u'821c6749-8fe8-4d6e-9ee6-5d47785dcc19', u'deviceName': u'Front Door', u'house': {u'houseName': u'Haist House', u'houseID': u'109c8131-f8e1-453a-89d1-35a59d79fb09'}, u'otherUser': {u'UserName': u'deleteduser', u'LastName': u'User', u'UserID': u'deleted', u'PhoneNo': u'deleted', u'FirstName': u'Unknown'}, u'dateTime': 1599492331114, u'entities': {u'device': u'F37A48FE11F140A4840E9552468A4945', u'house': u'109c8131-f8e1-453a-89d1-35a59d79fb09', u'callingUser': u'677f1a24-1d34-49ae-8f9b-d15db33c6a52', u'otherUser': u'deleted', u'activity': u'5f5650ece4272fe3bac2667e'}, u'callingUser': {u'LastName': u'Mackenzie', u'UserID': u'677f1a24-1d34-49ae-8f9b-d15db33c6a52', u'imageInfo': {u'original': {u'url': u'https://d33mytkkohwnk6.cloudfront.net/user/4d9992a0-3278-4179-bcd9-ba3a7c193fda.jpg', u'width': 2131, u'format': u'jpg', u'secure_url': u'https://d33mytkkohwnk6.cloudfront.net/user/4d9992a0-3278-4179-bcd9-ba3a7c193fda.jpg', u'height': 2131}, u'thumbnail': {u'url': u'https://d33mytkkohwnk6.cloudfront.net/user/aff607c9-292d-4b7b-b3e3-4589acee2447.jpg', u'width': 128, u'format': u'jpg', u'secure_url': u'https://d33mytkkohwnk6.cloudfront.net/user/aff607c9-292d-4b7b-b3e3-4589acee2447.jpg', u'height': 128}}, u'FirstName': u'Craig'}, u'info': {u'action': u'lock', u'dateTime': u'2020-09-07T15:25:31.114Z', u'DateLogActionID': u'F37A48FE11F140A4840E9552468A4945+Mon Sep 07 2020 15:25:31 GMT+0000 (Coordinated Universal Time)lock', u'crypto': u'success', u'mechanical': u'success'}, u'deviceType': u'lock', u'deviceID': u'F37A48FE11F140A4840E9552468A4945', u'action': u'lock'}

2020-09-07 12:39:52      August Home Debug               processing activity item: {u'info': {u'DateLogActionID': u'F37A48FE11F140A4840E9552468A4945+Mon Sep 07 2020 15:11:36 GMT+0000 (Coordinated Universal Time)doorclosed'}, u'deviceName': u'Front Door', u'house': {u'houseName': u'Haist House', u'houseID': u'109c8131-f8e1-453a-89d1-35a59d79fb09'}, u'otherUser': {u'UserName': u'deleteduser', u'LastName': u'User', u'UserID': u'deleted', u'PhoneNo': u'deleted', u'FirstName': u'Unknown'}, u'dateTime': 1599491496000, u'entities': {u'device': u'F37A48FE11F140A4840E9552468A4945', u'house': u'109c8131-f8e1-453a-89d1-35a59d79fb09', u'callingUser': u'deleted', u'otherUser': u'deleted', u'activity': u'5f564dc2b48c3309eb5cc3d5'}, u'callingUser': {u'UserName': u'deleteduser', u'LastName': u'User', u'UserID': u'deleted', u'PhoneNo': u'deleted', u'FirstName': u'Unknown'}, u'deviceType': u'lock', u'deviceID': u'F37A48FE11F140A4840E9552468A4945', u'action': u'doorclosed'}

2020-09-07 12:39:52      August Home Debug               processing activity item: {u'info': {u'DateLogActionID': u'F37A48FE11F140A4840E9552468A4945+Mon Sep 07 2020 15:11:19 GMT+0000 (Coordinated Universal Time)dooropen'}, u'deviceName': u'Front Door', u'house': {u'houseName': u'Haist House', u'houseID': u'109c8131-f8e1-453a-89d1-35a59d79fb09'}, u'otherUser': {u'UserName': u'deleteduser', u'LastName': u'User', u'UserID': u'deleted', u'PhoneNo': u'deleted', u'FirstName': u'Unknown'}, u'dateTime': 1599491479000, u'entities': {u'device': u'F37A48FE11F140A4840E9552468A4945', u'house': u'109c8131-f8e1-453a-89d1-35a59d79fb09', u'callingUser': u'deleted', u'otherUser': u'deleted', u'activity': u'5f564d9ebd4e2e385ba32a19'}, u'callingUser': {u'UserName': u'deleteduser', u'LastName': u'User', u'UserID': u'deleted', u'PhoneNo': u'deleted', u'FirstName': u'Unknown'}, u'deviceType': u'lock', u'deviceID': u'F37A48FE11F140A4840E9552468A4945', u'action': u'dooropen'}

2020-09-07 12:39:52      August Home Debug               processing activity item: {u'info': {u'DateLogActionID': u'F37A48FE11F140A4840E9552468A4945+Mon Sep 07 2020 15:11:18 GMT+0000 (Coordinated Universal Time)unlock'}, u'deviceName': u'Front Door', u'house': {u'houseName': u'Haist House', u'houseID': u'109c8131-f8e1-453a-89d1-35a59d79fb09'}, u'otherUser': {u'UserName': u'deleteduser', u'LastName': u'User', u'UserID': u'deleted', u'PhoneNo': u'deleted', u'FirstName': u'Unknown'}, u'dateTime': 1599491478000, u'entities': {u'device': u'F37A48FE11F140A4840E9552468A4945', u'house': u'109c8131-f8e1-453a-89d1-35a59d79fb09', u'callingUser': u'manualunlock', u'otherUser': u'deleted', u'activity': u'5f564d9db361e4ff46cbe4c6'}, u'callingUser': {u'LastName': u'Unlock', u'UserID': u'manualunlock', u'FirstName': u'Manual'}, u'deviceType': u'lock', u'deviceID': u'F37A48FE11F140A4840E9552468A4945', u'action': u'unlock'}

2020-09-07 12:39:52      August Home Debug               processing activity item: {u'info': {u'DateLogActionID': u'35FED6E34C3A40FE941CBEC085CE801A+Mon Sep 07 2020 14:01:20 GMT+0000 (Coordinated Universal Time)doorclosed'}, u'deviceName': u'Back Door', u'house': {u'houseName': u'Haist House', u'houseID': u'109c8131-f8e1-453a-89d1-35a59d79fb09'}, u'otherUser': {u'UserName': u'deleteduser', u'LastName': u'User', u'UserID': u'deleted', u'PhoneNo': u'deleted', u'FirstName': u'Unknown'}, u'dateTime': 1599487280000, u'entities': {u'device': u'35FED6E34C3A40FE941CBEC085CE801A', u'house': u'109c8131-f8e1-453a-89d1-35a59d79fb09', u'callingUser': u'deleted', u'otherUser': u'deleted', u'activity': u'5f563d54e4272f8108bd9a3b'}, u'callingUser': {u'UserName': u'deleteduser', u'LastName': u'User', u'UserID': u'deleted', u'PhoneNo': u'deleted', u'FirstName': u'Unknown'}, u'deviceType': u'lock', u'deviceID': u'35FED6E34C3A40FE941CBEC085CE801A', u'action': u'doorclosed'}

2020-09-07 12:39:52      August Home Debug               processing activity item: {u'info': {u'DateLogActionID': u'35FED6E34C3A40FE941CBEC085CE801A+Mon Sep 07 2020 14:01:17 GMT+0000 (Coordinated Universal Time)dooropen'}, u'deviceName': u'Back Door', u'house': {u'houseName': u'Haist House', u'houseID': u'109c8131-f8e1-453a-89d1-35a59d79fb09'}, u'otherUser': {u'UserName': u'deleteduser', u'LastName': u'User', u'UserID': u'deleted', u'PhoneNo': u'deleted', u'FirstName': u'Unknown'}, u'dateTime': 1599487277000, u'entities': {u'device': u'35FED6E34C3A40FE941CBEC085CE801A', u'house': u'109c8131-f8e1-453a-89d1-35a59d79fb09', u'callingUser': u'deleted', u'otherUser': u'deleted', u'activity': u'5f563d34677261486c124974'}, u'callingUser': {u'UserName': u'deleteduser', u'LastName': u'User', u'UserID': u'deleted', u'PhoneNo': u'deleted', u'FirstName': u'Unknown'}, u'deviceType': u'lock', u'deviceID': u'35FED6E34C3A40FE941CBEC085CE801A', u'action': u'dooropen'}

2020-09-07 12:39:52      August Home Debug               processing activity item: {u'info': {u'DateLogActionID': u'35FED6E34C3A40FE941CBEC085CE801A+Mon Sep 07 2020 14:00:08 GMT+0000 (Coordinated Universal Time)doorclosed'}, u'deviceName': u'Back Door', u'house': {u'houseName': u'Haist House', u'houseID': u'109c8131-f8e1-453a-89d1-35a59d79fb09'}, u'otherUser': {u'UserName': u'deleteduser', u'LastName': u'User', u'UserID': u'deleted', u'PhoneNo': u'deleted', u'FirstName': u'Unknown'}, u'dateTime': 1599487208000, u'entities': {u'device': u'35FED6E34C3A40FE941CBEC085CE801A', u'house': u'109c8131-f8e1-453a-89d1-35a59d79fb09', u'callingUser': u'deleted', u'otherUser': u'deleted', u'activity': u'5f563cea67726183361239bb'}, u'callingUser': {u'UserName': u'deleteduser', u'LastName': u'User', u'UserID': u'deleted', u'PhoneNo': u'deleted', u'FirstName': u'Unknown'}, u'deviceType': u'lock', u'deviceID': u'35FED6E34C3A40FE941CBEC085CE801A', u'action': u'doorclosed'}

2020-09-07 12:39:52      August Home Debug               processing activity item: {u'info': {u'DateLogActionID': u'35FED6E34C3A40FE941CBEC085CE801A+Mon Sep 07 2020 13:59:49 GMT+0000 (Coordinated Universal Time)dooropen'}, u'deviceName': u'Back Door', u'house': {u'houseName': u'Haist House', u'houseID': u'109c8131-f8e1-453a-89d1-35a59d79fb09'}, u'otherUser': {u'UserName': u'deleteduser', u'LastName': u'User', u'UserID': u'deleted', u'PhoneNo': u'deleted', u'FirstName': u'Unknown'}, u'dateTime': 1599487189000, u'entities': {u'device': u'35FED6E34C3A40FE941CBEC085CE801A', u'house': u'109c8131-f8e1-453a-89d1-35a59d79fb09', u'callingUser': u'deleted', u'otherUser': u'deleted', u'activity': u'5f563cdab48c3352ad58b3c6'}, u'callingUser': {u'UserName': u'deleteduser', u'LastName': u'User', u'UserID': u'deleted', u'PhoneNo': u'deleted', u'FirstName': u'Unknown'}, u'deviceType': u'lock', u'deviceID': u'35FED6E34C3A40FE941CBEC085CE801A', u'action': u'dooropen'}

2020-09-07 12:39:52      August Home Debug               processing activity item: {u'info': {u'DateLogActionID': u'35FED6E34C3A40FE941CBEC085CE801A+Mon Sep 07 2020 13:34:01 GMT+0000 (Coordinated Universal Time)doorclosed'}, u'deviceName': u'Back Door', u'house': {u'houseName': u'Haist House', u'houseID': u'109c8131-f8e1-453a-89d1-35a59d79fb09'}, u'otherUser': {u'UserName': u'deleteduser', u'LastName': u'User', u'UserID': u'deleted', u'PhoneNo': u'deleted', u'FirstName': u'Unknown'}, u'dateTime': 1599485641000, u'entities': {u'device': u'35FED6E34C3A40FE941CBEC085CE801A', u'house': u'109c8131-f8e1-453a-89d1-35a59d79fb09', u'callingUser': u'deleted', u'otherUser': u'deleted', u'activity': u'5f5636ccb361e46b61c69d01'}, u'callingUser': {u'UserName': u'deleteduser', u'LastName': u'User', u'UserID': u'deleted', u'PhoneNo': u'deleted', u'FirstName': u'Unknown'}, u'deviceType': u'lock', u'deviceID': u'35FED6E34C3A40FE941CBEC085CE801A', u'action': u'doorclosed'}

2020-09-07 12:39:52      August Home Debug               processing activity item: {u'info': {u'DateLogActionID': u'35FED6E34C3A40FE941CBEC085CE801A+Mon Sep 07 2020 13:33:31 GMT+0000 (Coordinated Universal Time)dooropen'}, u'deviceName': u'Back Door', u'house': {u'houseName': u'Haist House', u'houseID': u'109c8131-f8e1-453a-89d1-35a59d79fb09'}, u'otherUser': {u'UserName': u'deleteduser', u'LastName': u'User', u'UserID': u'deleted', u'PhoneNo': u'deleted', u'FirstName': u'Unknown'}, u'dateTime': 1599485611000, u'entities': {u'device': u'35FED6E34C3A40FE941CBEC085CE801A', u'house': u'109c8131-f8e1-453a-89d1-35a59d79fb09', u'callingUser': u'deleted', u'otherUser': u'deleted', u'activity': u'5f5636afb361e4dfe9c69783'}, u'callingUser': {u'UserName': u'deleteduser', u'LastName': u'User', u'UserID': u'deleted', u'PhoneNo': u'deleted', u'FirstName': u'Unknown'}, u'deviceType': u'lock', u'deviceID': u'35FED6E34C3A40FE941CBEC085CE801A', u'action': u'dooropen'}

2020-09-07 12:39:52      August Home Debug               processing activity item: {u'info': {u'DateLogActionID': u'35FED6E34C3A40FE941CBEC085CE801A+Mon Sep 07 2020 12:35:58 GMT+0000 (Coordinated Universal Time)doorclosed'}, u'deviceName': u'Back Door', u'house': {u'houseName': u'Haist House', u'houseID': u'109c8131-f8e1-453a-89d1-35a59d79fb09'}, u'otherUser': {u'UserName': u'deleteduser', u'LastName': u'User', u'UserID': u'deleted', u'PhoneNo': u'deleted', u'FirstName': u'Unknown'}, u'dateTime': 1599482158000, u'entities': {u'device': u'35FED6E34C3A40FE941CBEC085CE801A', u'house': u'109c8131-f8e1-453a-89d1-35a59d79fb09', u'callingUser': u'deleted', u'otherUser': u'deleted', u'activity': u'5f562951b361e432bac44f82'}, u'callingUser': {u'UserName': u'deleteduser', u'LastName': u'User', u'UserID': u'deleted', u'PhoneNo': u'deleted', u'FirstName': u'Unknown'}, u'deviceType': u'lock', u'deviceID': u'35FED6E34C3A40FE941CBEC085CE801A', u'action': u'doorclosed'}

2020-09-07 12:39:52      August Home Debug               processing activity item: {u'info': {u'DateLogActionID': u'35FED6E34C3A40FE941CBEC085CE801A+Mon Sep 07 2020 12:35:55 GMT+0000 (Coordinated Universal Time)dooropen'}, u'deviceName': u'Back Door', u'house': {u'houseName': u'Haist House', u'houseID': u'109c8131-f8e1-453a-89d1-35a59d79fb09'}, u'otherUser': {u'UserName': u'deleteduser', u'LastName': u'User', u'UserID': u'deleted', u'PhoneNo': u'deleted', u'FirstName': u'Unknown'}, u'dateTime': 1599482155000, u'entities': {u'device': u'35FED6E34C3A40FE941CBEC085CE801A', u'house': u'109c8131-f8e1-453a-89d1-35a59d79fb09', u'callingUser': u'deleted', u'otherUser': u'deleted', u'activity': u'5f56292fe4272fdcd4b9eae8'}, u'callingUser': {u'UserName': u'deleteduser', u'LastName': u'User', u'UserID': u'deleted', u'PhoneNo': u'deleted', u'FirstName': u'Unknown'}, u'deviceType': u'lock', u'deviceID': u'35FED6E34C3A40FE941CBEC085CE801A', u'action': u'dooropen'}

2020-09-07 12:39:52      August Home Debug               processing activity item: {u'info': {u'DateLogActionID': u'35FED6E34C3A40FE941CBEC085CE801A+Mon Sep 07 2020 12:26:13 GMT+0000 (Coordinated Universal Time)doorclosed'}, u'deviceName': u'Back Door', u'house': {u'houseName': u'Haist House', u'houseID': u'109c8131-f8e1-453a-89d1-35a59d79fb09'}, u'otherUser': {u'UserName': u'deleteduser', u'LastName': u'User', u'UserID': u'deleted', u'PhoneNo': u'deleted', u'FirstName': u'Unknown'}, u'dateTime': 1599481573000, u'entities': {u'device': u'35FED6E34C3A40FE941CBEC085CE801A', u'house': u'109c8131-f8e1-453a-89d1-35a59d79fb09', u'callingUser': u'deleted', u'otherUser': u'deleted', u'activity': u'5f5626eee4272f3007b995ea'}, u'callingUser': {u'UserName': u'deleteduser', u'LastName': u'User', u'UserID': u'deleted', u'PhoneNo': u'deleted', u'FirstName': u'Unknown'}, u'deviceType': u'lock', u'deviceID': u'35FED6E34C3A40FE941CBEC085CE801A', u'action': u'doorclosed'}

2020-09-07 12:39:52      August Home Debug               processing activity item: {u'info': {u'DateLogActionID': u'35FED6E34C3A40FE941CBEC085CE801A+Mon Sep 07 2020 12:20:58 GMT+0000 (Coordinated Universal Time)dooropen'}, u'deviceName': u'Back Door', u'house': {u'houseName': u'Haist House', u'houseID': u'109c8131-f8e1-453a-89d1-35a59d79fb09'}, u'otherUser': {u'UserName': u'deleteduser', u'LastName': u'User', u'UserID': u'deleted', u'PhoneNo': u'deleted', u'FirstName': u'Unknown'}, u'dateTime': 1599481258000, u'entities': {u'device': u'35FED6E34C3A40FE941CBEC085CE801A', u'house': u'109c8131-f8e1-453a-89d1-35a59d79fb09', u'callingUser': u'deleted', u'otherUser': u'deleted', u'activity': u'5f5625ade4272f5033b96a85'}, u'callingUser': {u'UserName': u'deleteduser', u'LastName': u'User', u'UserID': u'deleted', u'PhoneNo': u'deleted', u'FirstName': u'Unknown'}, u'deviceType': u'lock', u'deviceID': u'35FED6E34C3A40FE941CBEC085CE801A', u'action': u'dooropen'}

2020-09-07 12:39:52      August Home Debug               processing activity item: {u'info': {u'DateLogActionID': u'35FED6E34C3A40FE941CBEC085CE801A+Mon Sep 07 2020 12:20:57 GMT+0000 (Coordinated Universal Time)unlock'}, u'deviceName': u'Back Door', u'house': {u'houseName': u'Haist House', u'houseID': u'109c8131-f8e1-453a-89d1-35a59d79fb09'}, u'otherUser': {u'UserName': u'deleteduser', u'LastName': u'User', u'UserID': u'deleted', u'PhoneNo': u'deleted', u'FirstName': u'Unknown'}, u'dateTime': 1599481257000, u'entities': {u'device': u'35FED6E34C3A40FE941CBEC085CE801A', u'house': u'109c8131-f8e1-453a-89d1-35a59d79fb09', u'callingUser': u'manualunlock', u'otherUser': u'deleted', u'activity': u'5f5625ade4272faad9b96a79'}, u'callingUser': {u'LastName': u'Unlock', u'UserID': u'manualunlock', u'FirstName': u'Manual'}, u'deviceType': u'lock', u'deviceID': u'35FED6E34C3A40FE941CBEC085CE801A', u'action': u'unlock'}

2020-09-07 12:39:52      August Home Debug               Added 20 new Activity log items to the plugin internal cache

2020-09-07 12:39:52      August Home Debug               Current list of cache timestamp log items for the house (size: 20):

**Comment by mcraigm** [comment]
Hi Micheal,

 

I’m sorry, I had an issue with naming when I added the two new locks in.  They are both there and all four locks are working great.

 

Thanks, and again sorry for using up your time.

 

Cheers

Craig.

**Comment by mlamoure** [comment]
No worries.  Curious though, was there any issue where specific characters made it so that the locks didn't appear?

With four locks, are you able to consistently get state updates for all within 10 seconds or so of a change?

**Comment by mcraigm** [comment]
No it was totally my fault.  I changed the old locks to new locks and re-named the old locks.  When only 2 locks appeared in the New Device option It appeared like the old lock were still there and the new locks were not.  I was expecting to see 4 locks. – Again totally my fault!  

 

We are in a rural area and our internet is slow so I set the update to 90 seconds.  I haven’t set it lower but at 90 it works just fine.   I’m not sure if indigo checks the locks by WiFi on our network or if it checks the locks through the internet and I didn’t want a lot of internet use.

 

Thanks for your help
